src/main/java/com/epam/training/array/ArrayLogic.java

```diff
@@ -1,16 +1,72 @@
 package com.epam.training.array;
+import java.util.Arrays;
 
 public class ArrayLogic {
 
-    //TODO: write it using binary search
-    public boolean exists(Array array) {
-        throw new UnsupportedOperationException();
+    public boolean exists(Array array, int element) {
+        int res = Arrays.binarySearch(array.getItems(), element);
```

Comment:
@JK8789 Don't use util class `Arrays` please. You should implement your own logic

Reply:
Done